examples/ecs/iter_combinations.rs

```diff
@@ -54,10 +54,13 @@ fn generate_bodies(
     mut meshes: ResMut<Assets<Mesh>>,
     mut materials: ResMut<Assets<StandardMaterial>>,
 ) {
-    let mesh = meshes.add(Mesh::from(shape::Icosphere {
-        radius: 1.0,
-        subdivisions: 3,
-    }));
+    let mesh = meshes.add(
+        Mesh::try_from(shape::Icosphere {
+            radius: 1.0,
+            subdivisions: 3,
+        })
+        .expect("Error while creating mesh from icosphere"),
```

Comment:
```suggestion
        .unwrap(),
```

We have a nice error type; unwrapping will give better user output and is more maintainable.